docs/redoc-vendor-extensions.md

```diff
@@ -1,8 +1,61 @@
 # ReDoc vendor extensions
-ReDoc makes use of the following [vendor extensions](https://swagger.io/specification/#specificationExtensions)
 
-### Swagger Object vendor extensions
-Extend OpenAPI root [Swagger Object](https://swagger.io/specification/#oasObject)
+You can use the following [vendor extensions](https://swagger.io/specification/#specificationExtensions) with Redoc.
+
+<!-- TOC depthFrom:1 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [ReDoc vendor extensions](#redoc-vendor-extensions)
```

Comment:
For some reason this is rendering all in one line instead of like a nice TOC.

![image](https://user-images.githubusercontent.com/1161871/93411699-5cc2ae00-f861-11ea-89f6-502842f7839e.png)

Maybe there needs to be a space at the end of each line? (I'm not actually sure)